openapi-spec/swagger.yaml

```diff
@@ -0,0 +1,534 @@
+            $ref: '#/definitions/ServiceDefsReq'
+      responses:
+        '202':
+          description: Accepted
+          schema:
+            $ref: '#/definitions/ServiceDefs'
+        '400':
+          $ref: '#/responses/HTTPStatus400'
+        '401':
+          $ref: '#/responses/HTTPStatus401'
+        '403':
+          $ref: '#/responses/HTTPStatus403'
+        '500':
+          $ref: '#/responses/HTTPStatus500'
+  '/v1beta/{tenantId}/orchestration/instances':
+    parameters:
+      - $ref: '#/parameters/tenantId'
+    get:
+      tags:
+        - Service Instance
+      description: Gets all service instances detail.
+      responses:
+        '200':
+          description: OK
+          schema:
+            $ref: '#/definitions/InstanceSpecs'
+        '401':
+          $ref: '#/responses/HTTPStatus401'
+        '403':
+          $ref: '#/responses/HTTPStatus403'
+        '404':
+          $ref: '#/responses/HTTPStatus404'
+        '500':
+          $ref: '#/responses/HTTPStatus500'
+    post:
+      tags:
+        - Service Instance
+      description: Create an instance of service definition.
+      parameters:
+        - name: Instance details
+          in: body
+          schema:
+            $ref: '#/definitions/InstanceReq'
+      responses:
+        '200':
+          description: OK
+          schema:
+            $ref: '#/definitions/InstanceCreationResp'
+        '401':
+          $ref: '#/responses/HTTPStatus401'
+        '403':
+          $ref: '#/responses/HTTPStatus403'
+        '404':
+          $ref: '#/responses/HTTPStatus404'
+        '500':
+          $ref: '#/responses/HTTPStatus500'
+    delete:
+      tags:
+        - Service Instance
+      description: Delete an instance in execution
+      parameters:
+        - name: 'Execution details'
+          in: body
+          schema:
+            $ref: '#/definitions/InstanceCreationResp'
+      responses:
+        '200':
+          description: OK
+          schema:
+            $ref: '#/definitions/InstanceCreationResp'
+        '401':
+          $ref: '#/responses/HTTPStatus401'
+        '403':
+          $ref: '#/responses/HTTPStatus403'
+        '404':
+          $ref: '#/responses/HTTPStatus404'
+        '500':
+          $ref: '#/responses/HTTPStatus500'
+  '/v1beta/{tenantId}/orchestration/instances/{instanceId}':
+    parameters:
+      - $ref: '#/parameters/tenantId'
+      - $ref: '#/parameters/instanceId'
+    get:
+      tags:
+        - Service Instance
+      description: Gets particular service instance detail.
+      responses:
+        '200':
+          description: OK
+          schema:
+            $ref: '#/definitions/InstanceSpecs'
+        '401':
+          $ref: '#/responses/HTTPStatus401'
+        '403':
+          $ref: '#/responses/HTTPStatus403'
+        '404':
+          $ref: '#/responses/HTTPStatus404'
+        '500':
+          $ref: '#/responses/HTTPStatus500'
+  '/v1beta/{tenantId}/orchestration/tasks/{executionId}':
+    parameters:
```

Comment:
the api looks weird, is it task list api ? I think there two api about task api: list tasks and show a task

Reply:
This is show execution task